### PR TITLE
feat: add getting started

### PR DESCRIPTION
**Pull request by AnjaMis**
- Added getting started section
- Reorganized the navigation again
- Edited Content

## Summary by CodeRabbit

- **New Features**
  - Introduced a "Getting Started" section for new users, enhancing navigation and accessibility.
  - Added a "Quickstart Guide" section for streamlined onboarding.

- **Documentation Updates**
  - Revised various documentation sections to reflect current statuses, including "Work in Progress ⚙️".
  - Updated hyperlinks for better accuracy and navigation, transitioning from absolute to relative paths.

- **Bug Fixes**
  - Removed redundant quiz components from multiple sections, simplifying content presentation.

- **Style Improvements**
  - Enhanced clarity in titles and descriptions throughout the documentation.

- **Chores**
  - Updated navigation labels and links to reflect current branding and community resources.
  - Adjusted ordering of documentation sections for improved user experience.

### CODE REVIEW

File: docs/getting-started/introduction/protocol-concepts.md [hunk 1]
```diff
@@ -45,8 +45,8 @@ As a decentralized orchestration layer for off-chain resources, Axone must ensur
 - Interpret the rules to ensure the law enforcement
 - Execute workflow and orchestrate the resources according to the rules
 
-In [Rules](https://docs.okp4.network/introduction/rules) section, we describe how the protocol allows you to **define and enforce any customized rules**.
-In [Workflows](https://docs.okp4.network/introduction/workflows) section, we describe how the protocol **orchestrates the resources according to the rules**.
+In [Rules](https://docs.okp4.network/getting-started/introduction/rules) section, we describe how the protocol allows you to **define and enforce any customized rules**.
```
Comment (ccamel) [inline docs/getting-started/introduction/protocol-concepts.md:48]: Using relative URLs is preferable for denoting internal links (see: [markdown-features/links](https://docusaurus.io/docs/markdown-features/links)). Would be advisable to ensure that all internal links follow this format.

```suggestion
In [Rules](./getting-started/introduction/rules) section, we describe how the protocol allows you to **define and enforce any customized rules**.
```

**Review by ccamel — CHANGES_REQUESTED** [review]
Minor remark. If you could consider it, we’ll be good. 😌  Thanks.

File: docs/faq/faq.md [hunk 2]
```diff
@@ -72,7 +72,7 @@ An ontology, in the context of Axone, refers to a structured representation of k
 
 In Axone, this ontology is used to describe the metadata or characteristics of resources and services. Think of it as the "data about the data." This structured information makes it easier for users and systems to understand and interact with resources and services within the Axone network. It ensures that everyone speaks the same language when it comes to sharing and accessing digital resources.
 
-Dedicated documentation [here](https://docs.okp4.network/whitepaper/solution#ontology)
+Dedicated documentation [here](./whitepaper/solution#ontology)
 
 And deep-dive article [here](https://blog.okp4.network/okp4-knowledge-sharing-through-ontology-driven-dataverse-a-new-approach-to-knowledge-description-f9b81d4484c8)
```
Comment (amimart) [inline docs/faq/faq.md:77]:
```suggestion
And deep-dive article [here](https://blog.axone.xyz/okp4-knowledge-sharing-through-ontology-driven-dataverse-a-new-approach-to-knowledge-description-f9b81d4484c8)
```

This is not part of the changes but I noticed this wrong URL

File: docs/getting-started/quickstart-guides/describe-resource.mdx [hunk 3]
```diff
@@ -114,7 +112,7 @@ In the future, you will be able to use any key provider, such as Cheqd.
 
 ## Step 2: Create the credentials
 
-Find the credential templates you need in the [Ontology documentation](https://docs.okp4.network/ontology/schemas) according to the nature of the resource you describe.
+Find the credential templates you need in the [Ontology documentation](./ontology/schemas) according to the nature of the resource you describe.
 
 For the dataset, instantiate the template [credential-dataset-description](https://github.com/okp4/ontology/blob/main/src/example/dataset/crime-dataset-description.jsonld).
```
Comment (amimart) [inline docs/getting-started/quickstart-guides/describe-resource.mdx:117]:
```suggestion
For the dataset, instantiate the template [credential-dataset-description](https://github.com/axone-protocol/ontology/blob/main/src/example/dataset/crime-dataset-description.jsonld).  
```

**Review by amimart — APPROVED** [review]
Looks good :)

I noticed old links towards `okp4` URLs or GitHub organization, I'll let you juge if it enters the scope of this PR

**Review by ccamel — APPROVED** [review]
Looks good 👍

**Pull request merged**